.devcontainer/Dockerfile

```diff
@@ -50,7 +52,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         network-manager \
         apparmor-utils \
         libpulse0 \
-    && bash <(curl https://getvcn.codenotary.com -L) \
+    && curl -Lo /bin/vcn https://github.com/codenotary/vcn/releases/download/${VCN_VERSION}/vcn-${VCN_VERSION}-linux-amd64-static \
```

Comment:
We should use the same approach in the main `Dockerfile`

Reply:
we can't because of all archs